hack/tools/README.md

```diff
@@ -0,0 +1,5 @@
+This directory contains a stub go module used to track version of development
```

Comment:
we compile tools in a separate module, then our project module will not be touched